live-examples/css-examples/flexbox/flex-shrink.html

```diff
@@ -0,0 +1,36 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="flex-shrink">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">flex-shrink: 1;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">flex-shrink: 2;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">flex-shrink: 3;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    
+
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example" class="default-example">
+        <div class="example-container">
+            <div id="example-element" class="transition-all">Item One</div>
```

Comment:
I am wondering, could we title this one as "I shrink"? Makes it clearer that the `flex-shrink` rule is applied to this specific element.